src/Centreon/Domain/Monitoring/ResourceService.php

```diff
@@ -91,8 +92,8 @@ public function findResources(ResourceFilter $filter): array
         // try to avoid exception from the regexp bad syntax in search criteria
         try {
             $list = $this->resourceRepository->findResources($filter);
-        } catch (ResourceRegExpException $exception) {
-            $list = [];
+        } catch (ResourceException $ex) {
```

Comment:
Why you catch and after that throw the same exception?

Reply:
It's my bad. Not a good idea to code at 2:00 am